2-mandatory/4-restaurant.js

```diff
@@ -21,6 +21,18 @@ const MENU = {
 
 let cashRegister = {
   // write code here
+  orderBurger: (balance) => {
+    if (balance - MENU.burger >= 0) {
+      balance -= MENU.burger;
+    }
+    return balance;
+    },
+  orderFalafel: (balance) => {
+    if (balance - MENU.falafel >= 0) {
+      balance -= MENU.falafel;
+    }
+    return balance;
+  }
```

Comment:
Looks, good, but can you also think about how to achieve the same result with inline arrow functions?